app/javascript/packages/analytics/index.js

```diff
@@ -0,0 +1,20 @@
+import { loadPolyfills } from '@18f/identity-polyfill';
+
+const LOGGER_ENDPOINT = '/api/logger';
+
+/**
+ * Logs an event.
+ *
+ * @param {string} event Event name.
+ * @param {object=} payload Payload object.
+ *
+ * @return {Promise<Response>}
+ */
+export async function trackEvent(event, payload) {
+  await loadPolyfills(['fetch']);
```

Comment:
Realizing it's kinda inconsistent that we load polyfill in this package, but then for `document-capture-poling` it's assumed to already be polyfilled by the time the class is constructed.

Reply:
> Realizing it's kinda inconsistent that we load polyfill in this package, but then for `document-capture-poling` it's assumed to already be polyfilled by the time the class is constructed.

I'll plan to remove this, I think. Not sure if I love either of the options of loading the polyfill at the `packs` level vs. at individual packages where they're required, but it seems we're currently most consistent to keep them at `packs`.

Reply:
I agree with that. If we need to come back and change it for efficiency we can do that if it becomes an issue. Consistency seems like the best choice right now.

Reply:
Removed in 12011435b.